frontend/src/components/auth/ResetPasswordForm.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+    <VContent>
+        <VFlex
+                lg6
+                md6
+                sm12
+                xs12
+        >
+            <VContainer>
+                <VCardText>
+                    <VSubheader class="reset-password-form-header">
+                        Welcome to Metrica!
+                    </VSubheader>
+                    <VForm ref="form">
+                        <VSubheader class="reset-password-form-label">
+                            Email
+                        </VSubheader>
+                        <VTextField
+                                name="email"
+                                class="reset-password-form-input"
+                                v-model="email"
+                                solo
+                                type="text"
+                                :rules="emailRules"
+                                required
+                        />
+                    </VForm>
+                </VCardText>
+                <VCardActions>
+                    <VBtn @click="onResetPassword" class="reset-password-form-button mt-3" color="#3C57DE">
```

Comment:
ensure that your code uses styleguide suggestions, where even binding should places last one